rules/bus-test/rule.md

```diff
@@ -0,0 +1,57 @@
+---
+type: rule
+title: Does your project pass the bus test?
+uri: bus-test
+authors:
+  - title: Matt Goldman
+    url: https://ssw.com.au/people/matt-goldman
+  - title: Matt Wicks
+    url: https://ssw.com.au/people/matt-wicks
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+related:
+    - track-project-documents
+redirects:
+  - do-you-know-how-to-handover-a-project
+created: 2024-01-05T06:22:03.000Z
+archivedreason: null
+guid: f6d7fe16-0723-4f6f-8fd9-8e9ef1a0c289
+---
+
+We often talk about the 'bus test' to assess our project's resilience: What happens if your project lead gets hit by a bus? But let's consider a less gloomy, yet equally unexpected scenario: What if they're recruited for a top-secret government mission? In either case, the question remains: Is your project structured well enough to survive such unforeseen events?
+
+<!--endintro-->
+
+`youtube: https://www.youtube.com/embed/BIDErCLM7iA?rel=0`
+
+## Document Everything
+
+Ensure all project-related knowledge is well documented. This includes not only technical documentation but also process workflows, login credentials, and decision-making rationales. Use tools like Confluence to keep documentation centralized and accessible.
```

Comment:
I think this is great for now! 

But should we later add more to this in what we recommend as the best tools to keep documentation. i.e. we use SharePoint more than anything or DevOps Wiki etc

Reply:
That's a good point. There's a reference in the section on making sure documentation is accessible to the rule on keeping things in Teams.